packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -254,6 +254,23 @@ async function getOutlookExePath(): Promise<string> {
   }
 }
 
+async function getWXPExePath(app: OfficeApp): Promise<string> {
+  try {
+    const InstallPathRegistryKey: string = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\${app}.EXE`;
```

Comment:
Note that the exe names are not the same as the "app" name (Word is actually 'winword.exe').  Case sensitivity would give you problems as well ('excel.exe' != "Excel.exe".

You'll need some sort of mapping for that.  Maybe that means making a map in one of the package were the value is an object with all the information for a given host. . . like host title, exe name, etc.  It's something to think about beyond a switch just for this method.